tests/foreman/maintain/test_service.py

```diff
@@ -183,6 +188,28 @@ def test_positive_service_enable_disable(sat_maintain):
     result = sat_maintain.cli.Service.enable()
     assert 'FAIL' not in result.stdout
     assert result.status == 0
+    sat_maintain.power_control(state='reboot')
+    if type(sat_maintain) is Satellite:
```

Comment:
`isinstance` is prefered over `type` for type checking.
```suggestion
    if isinstance(sat_maintain, Satellite):
```